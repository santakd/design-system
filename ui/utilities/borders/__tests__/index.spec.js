/* eslint-env jest */
import React from 'react';
import createHelpers from '../../../../jest.helpers';
const { matchesMarkup } = createHelpers(__dirname);

describe('renders border utilities', () => {
  it('renders top border', () =>
    matchesMarkup(
      <div className="slds-border_top">This should have a top border.</div>
    ));

  it('renders right border', () =>
    matchesMarkup(
      <div className="slds-border_right">This should have a right border.</div>
    ));

  it('renders bottom border', () =>
    matchesMarkup(
      <div className="slds-border_bottom">
        This should have a bottom border.
      </div>
    ));

  it('renders left border', () =>
    matchesMarkup(
      <div className="slds-border_left">This should have a left border.</div>
    ));
});
