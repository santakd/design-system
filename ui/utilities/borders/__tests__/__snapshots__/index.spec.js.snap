// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders border utilities renders bottom border 1`] = `
<div
  className="slds-border_bottom"
>
  This should have a bottom border.
</div>
`;

exports[`renders border utilities renders left border 1`] = `
<div
  className="slds-border_left"
>
  This should have a left border.
</div>
`;

exports[`renders border utilities renders right border 1`] = `
<div
  className="slds-border_right"
>
  This should have a right border.
</div>
`;

exports[`renders border utilities renders top border 1`] = `
<div
  className="slds-border_top"
>
  This should have a top border.
</div>
`;
